### DISCUSSION TITLE
Projects Beta: Make status label colored

### DISCUSSION THREAD

**Discussion by seclerp** [community/community#9348] [Projects and Issues] [answered]
For now, it's not so easy to visually parse the status column in table view because all statuses have the same color:

![image](https://user-images.githubusercontent.com/20597871/147515377-fbb0e137-b01b-423b-a658-d8de4e3fb8f4.png)

Making them colorable will heavily improve UX in large boards

**Comment by heytdep** [comment]
This would be a great feature. Right now on my project I really needed this so I set up a simple snippet to run on the console to color our Statuses, pasting it here in case it may be useful to some users:

```javascript
const statuses = document.getElementsByClassName("Label-sc-6dyj7v-0")

const progress_colors = {
	"Done": "green",
	"Todo": "pink",
	"In Progress": "yellow"
}

for (let i=0; i < statuses.length; i++) {
	const status = statuses[i]
	const color = progress_colors[status.textContent]
	statuses[i].style.color = color
}
```

This is not an ideal solution but works fine. Anyways, pasting the above code in your console should show something like this:
![image](https://user-images.githubusercontent.com/70587974/147991315-64f04481-aed0-4758-b792-b64251a06ee1.png)

**Reply by seclerp** [reply]
Great job and thanks for sharing!

I also wanted to do something like that, but an out-of-the-box solution will be nice to have.

**Comment by j0siepy** [comment, marked as answer]
Hi folks!! Thank you so much for your patience - we've made this possible! Let us know if you have any future feedback for us ❤️

https://user-images.githubusercontent.com/98360703/217907748-9274f64d-fd19-4191-b855-6d0939fdf1ae.mov

**Comment by Jack-McKalling** [comment]
Please also render these colours on issue and PR timelines, similar to labels.

Such as in this type of entry:

> **github-project-automation** moved this to Options in # Features on Feb 10

Or this one:

> **myUsername** moved this from To Do to On hold (awaiting input) in # Maintenance 2w ago

(redacted hyperlinks from examples)
These full sentences can become difficult to read without further distinction. Now that there are colour associations, they might as well be used wherever their strings are displayed.

Let me know if I need to make a new post for this.
